apps/api-extractor/package.json

```diff
@@ -45,6 +45,7 @@
     "@rushstack/ts-command-line": "workspace:*",
     "colors": "~1.2.1",
     "lodash": "~4.17.15",
+    "minimatch": "~3.0.3",
```

Comment:
@octogonz At least one other library in this repo was already using minimatch at this version. Are we okay adding this dependency here? We can always write our own glob handler, but this seemed simpler if we're okay with the dependency.

Reply:
I think `minimatch` is okay. 👍